.gitignore

```diff
@@ -1,3 +1,8 @@
 .sass-cache/
 application.css
 application.css.map
+styles.css
+styles.css.map
+equalizer.css
+equalizer.css.map
```

Comment:
I think you only need `application` ones here. Remove both `styles.css and styles.css.map` and the ones for equalizer